pkgs/development/node-packages/composition-v13.nix

```diff
@@ -2,7 +2,7 @@
 
 {pkgs ? import <nixpkgs> {
     inherit system;
-  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-13_x"}:
+  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-8_x"}:
```

Comment:
I'm quite worried about this change.  It seems strange that `generate.sh` would replace `pkgs.nodejs-13_x` with `pkgs.nodejs-8_x` in this file (since it is composition-v_13_.nix).

I can remove this change from this PR if necessary.

Reply:
this seems like unwanted change to me as well

Reply:
Yeah, I guess the real question is, is this a problem in `node2nix` (and should be reported upstream), or is this a problem in the `generate.sh` script, and a new issue opened here on the nixpkgs repo.